### ISSUE TITLE
Minikube should return same IP address after restarting when using hyperkit.

### ISSUE THREAD

**Issue by bpasson** [kubernetes/minikube#2416]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
FEATURE REQUEST

Please provide the following details:

**Environment**:

**Minikube version** (use `minikube version`): 0.24.1
- **OS** (e.g. from /etc/os-release): OSX 10.11.6
- **VM Driver**: hyperkit
- **ISO version**: v0.24.0
- **Install tools**:
- **Others**:

**What happened**:
After creating a minikube using `minikube start --vm-driver=hyperkit`, the minikube gets assigned an IP address. I then stop it and start it again. The minikube gets assigned a different new IP address. 

**What you expected to happen**:
I would expect the minikube to have the same IP address after restarting the minikube.

**How to reproduce it** (as minimally and precisely as possible):
Run the following commands:
```
$ minikube start --vm-driver=hyperkit
$ minikube stop
$ minikube start
```

**Output of `minikube logs` (if applicable)**:

**Anything else do we need to know**:
After some investigation I found the hyperkit driver to generate a new random mac-address on every start of the machine. It does this based on a random UUID it generates. Saving the UUID to the configuration and using it upon starting the machine should solve the issue as the same mac address gets generated and thus gets assigned the same IP using DHCP.

Update: I managed to get it working for the hyperkit driver and will submit a pull request.

**Comment by dlorenc** [comment]
This should be fixed with #2417

**Comment by dlorenc** [comment]
D'oh, #2417 fixed this.
